soroban-env-host/src/test/storage.rs

```diff
@@ -397,6 +398,69 @@ fn test_nested_bump() {
     .unwrap()
 }
 
+#[test]
+fn test_muxed_account_is_not_allowed_as_storage_key() {
+    // let host = observe_host!(Host::test_host_with_recording_footprint());
```

Comment:
Nit: commented code

Reply:
Yeah, removed. FWIW I've got some weird compilation errors with observed host and didn't feel like observing this test is too critical.